sdk/src/proto/network.rs

```diff
@@ -89,15 +92,15 @@ pub struct ClaimProofResponse {
 #[allow(clippy::derive_partial_eq_without_eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct FulfillProofRequest {
+    /// The signature of the message.
```

Comment:
moved signature to first param in all of these because so it is safer to append fields later